src/csharp/Extensions/Microsoft.Spark.Extensions.DotNet.Interactive/AssemblyKernelExtension.cs

```diff
@@ -103,7 +103,7 @@ private bool TryEmitAssembly(CSharpKernel kernel, string dstPath, out string ass
 
         private bool TryGetSparkSession(out SparkSession sparkSession)
         {
-            sparkSession = SparkSession.GetDefaultSession();
+            sparkSession = SparkSession.GetActiveSession();
```

Comment:
Should we update this to `GetActiveSession` or keep it as `GetDefaultSession` ?

Reply:
Is this called only during the load? Then it can be either one, I think.